src/main/java/frc/robot/RobotContainer.java

```diff
@@ -359,7 +362,88 @@ public void autoChooserUpdate() {
                 new PositionIntake(m_elevator, m_wrist, () -> false, Position.STOW),
                 new WaitCommand(1.5),
                 new MoveElevator(m_elevator, ElevatorConstants.kAutoTop))),
-              new AutoDeposit(Position.TOP, m_elevator, m_wrist, m_intake, () -> false)));
+              new AutoDeposit(Position.TOP, true, m_elevator, m_wrist, m_intake, () -> false)));
+
+      m_autoCommand.addOption("ROUTINE 25: Grid 1 Two Piece Cube Top Engage",
+          Commands.sequence(
+              new AutoDeposit(Position.TOP, true, m_elevator, m_wrist, m_intake),
+              new PathPlannerCommand("Grid 1 Two Piece", 0, m_drive, true)
+                .alongWith(new WaitCommand(1.75).andThen(
+                  new PositionIntake(m_elevator, m_wrist, () -> false, Position.INTAKE), 
+                  new IntakeGamePiece(m_intake, () -> false, false))),
+              new PathPlannerCommand("Grid 1 Two Piece", 1, m_drive, true).deadlineWith(Commands.sequence(
+                new PositionIntake(m_elevator, m_wrist, () -> false, Position.STOW),
+                new WaitCommand(1.5),
+                new MoveElevator(m_elevator, ElevatorConstants.kAutoTop))),
+              new AutoDeposit(Position.TOP, true, m_elevator, m_wrist, m_intake, () -> false),
+              new PathPlannerCommand("Grid 1 Two Piece Engage", 0, m_drive, true),
+              new BalanceCommand(m_drive)));
```

Comment:
there is a two piece grid 9 but no two piece grid 1

Reply:
future PR? Grid one would require vision because it goes over the cable tray. It could mean substantial vision adjustments outside the scope of this PR.